packages/l/libmpdclient/xmake.lua

```diff
@@ -8,6 +8,8 @@ package("libmpdclient")
     on_install("linux", function (package)
         import("package.tools.meson").install(package)
         os.cp("include", package:installdir())
+        os.cp("build_*/version.h", package:installdir() .. "/include/mpd")
```

Comment:
import("package.tools.meson").install(package, {}, {buildir = "build"})
then 

os.cp("build/version.h" ...

Reply:
I can't use `build` as that folder is already present with files in it.
So I did that using `build_real` instead but it fails to build.

Because even though `ninja` is trying to use the `build_real` dir, meson still uses `build_$HASH`.

Reply:
oh, this is xmake's issue. I have fixed it on dev branch.

Reply:
All right, will install xmake from the dev branch then.

Reply:
we can improve it again after the new version is released.

Reply:
All right :+1: